lib/cmdledit.g

```diff
@@ -751,10 +751,108 @@ GAPInfo.CommandLineEditFunctions.Functions.(INT_CHAR(' ')) :=
                 GAPInfo.CommandLineEditFunctions.Functions.SpaceDeletePrompt;
 BindKeysToGAPHandler(" ");
 
+
+BindGlobal("STANDARD_EXTENDERS", rec(
+  caseSensitive := function(cand, word)
+      local i, j, c;
+      i := Length(word);
+      while true do
+        if i = Length(cand[1]) then
+          break;
+        fi;
+        c := cand[1][i+1];
+        for j in [2..Length(cand)] do
+          if Length(cand[j]) > i and cand[j][i+1] = c then
+            j := j+1;
+          else
+            break;
+          fi;
+        od;
+        if j <= Length(cand) then
+          break;
+        else
+          i := i+1;
+        fi;
+      od;
+      if i > Length(word) then
+        return cand[1]{[1..i]};
+      else
+        return fail;
+      fi;
+    end,
+
+    caseInsensitive := function(cand, word)
+      local i, j, c, lowword, filtequal;
+      # Check if exactly 'word' exists, ignoring case.
+      lowword := LowercaseString(word);
+      filtequal := Filtered(cand, a -> LowercaseString(a) = lowword);
+      # If there are several equal words, just pick the first one...
+      if Length(filtequal) > 0 then
+        return filtequal[1];
+      fi;
+      i := Length(word);
+      while true do
+        if i = Length(cand[1]) then
+          break;
+        fi;
+        c := LowercaseChar(cand[1][i+1]);
+        for j in [2..Length(cand)] do
+          if Length(cand[j]) > i and LowercaseChar(cand[j][i+1]) = c then
+            j := j+1;
+          else
+            break;
+          fi;
+        od;
+        if j <= Length(cand) then
+          break;
+        else
+          i := i+1;
+        fi;
+      od;
+      if i >= Length(word) then
+        return cand[1]{[1..i]};
+      else
+        return fail;
+      fi;
+    end
```

Comment:
perhaps add a comma after this, so that if more functions are added to the list in the future, no diff for that line shows up?